packages/manager/modules/pci/src/projects/project/kubernetes/components/gateway-management/constants.js

```diff
@@ -0,0 +1,5 @@
+export const GATEWAY_IP_REGEX = /^([0-9]{1,3}\.){3}[0-9]{1,3}?$/;
```

Comment:
```suggestion
export const GATEWAY_IP_REGEX = /^((25[0-5]|(2[0-4]|1\d|[1-9]|)\d)(\.(?!$)|$)){4}$/;
```
 Allows not to put ip of the type:
- 999.999.999.999
- 010.01.001.001

Reply:
It was here Guillaume.
It's not part of my dev.

Reply:
Thanks for 👍